### DISCUSSION TITLE
Translations are being updated?

### DISCUSSION THREAD

**Discussion by Pedro-vk** [melianmiko/OpenFreebuds#34] [Q&A] [answered]
Hi! I would like to help with the Spanish translation. Just wondering if you are still updating the app and if you are going to include the translations :)

Thanks for this fantastic project!

**Comment by melianmiko** [comment]
Hi,

New version of OpenFreebuds will be released next week, fresh strings to translate will be published in a few days.

**Comment by Pedro-vk** [comment, marked as answer]
> Hi,
> 
> New version of OpenFreebuds will be released next week, fresh strings to translate will be published in a few days.

Cool, I'll take a look next week and try to translate as much as possible to Spanish :+1:
